cmd/api-router.go

```diff
@@ -34,7 +34,7 @@ func registerAPIRouter(mux *router.Router) {
 	}
 
 	// API Router
-	apiRouter := router.NewRouter().PathPrefix("/").Subrouter()
+	apiRouter := mux.NewRoute().PathPrefix("/").Subrouter()
```

Comment:
This has bad effect on the handlers per route below (time validity handler), but looks like it works like this:

```
apiRouter := mux.NewRoute().PathPrefix("").Subrouter()
```

Isn't this more correct ? especially '/' is already added in bucket router.